src/components/RecipientAddressRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+import { HStack, Icon, Link } from '@chakra-ui/react'
+import type { Asset } from '@shapeshiftoss/types'
+
+import { Row } from '@/components/Row/Row'
+import { RawText, Text } from '@/components/Text'
+import { middleEllipsis } from '@/lib/utils'
+
+type RecipientAddressRowProps = {
+  sellAsset: Asset
+  recipientAddress: string | undefined
```

Comment:
Now that this is a dumb component, can we use this as an opportunity to make this required (remove the opt chain below too).

Consumers should guard against rendering it.

```suggestion
  recipientAddress: string
```

Another option is to keep typings as-is and early return, though preference for the first option, at this stage we *should* have a receive address.